src/Controls/src/Core/Picker/Picker.cs

```diff
@@ -278,9 +278,7 @@ void OnItemsSourceChanged(IList oldValue, IList newValue)
 			}
 			else
 			{
-				// Unlock, then clear, so OnItemsCollectionChanged executes
 				((LockableObservableListWrapper)Items).IsLocked = false;
-				((LockableObservableListWrapper)Items).InternalClear();
```

Comment:
According to the comment, we have to clear in order to execute OnItemsCollectionChanged . Has something changed or was the comment wrong?